feature-runner/steps/device.ts

```diff
@@ -43,14 +43,15 @@ const createDeviceForModel =
 			Type.Object({
 				model: Type.String(),
 				storageName: Type.String(),
+				account: Type.String(),
 			}),
 		)(
-			/^I have the fingerprint for a `(?<model>[^`]+)` device in `(?<storageName>[^`]+)`$/,
+			/^I have the fingerprint under `(?<account>[^`]+)` account for a `(?<model>[^`]+)` in `(?<storageName>[^`]+)`$/,
```

Comment:
```suggestion
			/^I have the fingerprint under `(?<account>[^`]+)` account for a `(?<model>[^`]+)` in `(?<storageName>[^`]+)`$/,
			/^I have the fingerprint for a `(?<model>[^`]+)` device(?<maybeAccount> in the `(?<account>[^`]+)` account) in `(?<storageName>[^`]+)`$/,
```

No need to change the other tests, use the default account.